### ISSUE TITLE
Need Information About SocketIO Proccess Request

### ISSUE THREAD

**Issue by oralunal** [PokemonGoF/PokemonGo-Bot#5035]
Dear Bot Developers,

I'm developping a WebUI for this bot. I can listen events via websocket. My question is can I emit command to bot? I looked for the files (I really don't understan Python) and thought I have emit command like this:

`socket_io.emit('bot:process_request:username', {'command':'release_pokemon', data: {'pokemon_id': 10000000}});`

But it didn't work. No one doesn't have information or doesn't interest on slack server.

Does bot access remote control?
If yes what commands does it access? How does it access?

**Comment by rawgni** [comment]
you should try to emit:
`remote:send_request, {'account': 'ptc/google account', 'name': 'get_player_info'`

and wait on
`bot:send_reply`

**Comment by sohje** [comment]
I think only get_player_info implemented.

**Comment by jannickv** [comment]
I can see the Polyline walker isn't emitting anything to the event_manager (in contrast to the step_walker). So i suppose it wouldn't log to terminal as well...

Think this should be a separate issue.

**Comment by Jasperrr91** [comment]
Hi @oralunal,

Not sure how far you are with developing but please feel free to continue on this base: https://github.com/PokemonGoF/PokemonGo-Web/tree/rewrite. It's an Angular-based web ui that uses the websockets for data.

Like @sohje said, only get_player_info is the only remote control command currently implemented. If you want other remote control commands, feel free to request them.

I'll look into the polylinewalker.

**Comment by oralunal** [comment]
Actually get_player_info is not working
